modules/admin_manual/pages/enterprise/reporting/metrics.adoc

```diff
@@ -131,3 +131,9 @@ curl -H "OC-MetricsApiKey: <your-metrics-secret>"  \
 
 TIP: Please replace `<your-metrics-secret>` with your respective system config value and `<your owncloud>`
 with the URL of your ownCloud instance.
+
+== Limitations
+
+Metrics app was desined for small deployments of ownCloud up to 250 Users.
+On a big instance with more than 250 users it can take a very long time to gather the data.
+You can speed this up by excluding userData and quota. 
```

Comment:
```suggestion
The Metrics app was desined for ownCloud deployments up to 250 users.
On deployments with more than 250 users, it can take considerabe longer to gather the requested data.
To reduce the time needed, you can exclude _userData_ and _quota_. 
```